server/index.js

```diff
@@ -2,22 +2,25 @@ import { ApolloServer } from 'apollo-server-express'
 import { importSchema } from 'graphql-import'
 import express from 'express'
 import path from 'path'
+import redis from 'async-redis'
 import resolvers from './resolvers'
-import { PORT } from './config'
+import { PORT, REDIS_IP, REDIS_PORT } from './config'
 
 const typeDefs = importSchema(path.join(__dirname, './typedefs/index.graphql'))
 
+const cache = redis.createClient({ host: REDIS_IP, port: REDIS_PORT })
+
 const server = new ApolloServer({
   typeDefs,
   resolvers,
-  context: ({ req = {}, res }) => ({ req, res }),
+  context: ({ req = {}, res }) => ({ req, res, cache }),
```

Comment:
nit: consider defaulting to no cache when the `REDIS_IP` is not specified. This way, a developer can still test locally without having to spin up their own Redis instance.